gnd/src/main/java/com/google/android/gnd/persistence/local/LocalDataStore.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 Google LLC
+ * Copyright 2021 Google LLC
```

Comment:
Please keep original copyright date on files. Optionally, you could write `Copyright 2019-2012...`

Same throughout.

Reply:
Whoops, for some reason I thought we needed to update theses. Fixed! 5ed2f340